app/components/ui/Navigation.module.scss

```diff
@@ -3,7 +3,7 @@
 @mixin siteNavShared {
   position: sticky;
   top: 0;
-  z-index: 1;
+  z-index: 3;
```

Comment:
I don't have a great suggestion for this right now, but I usually view it as a code smell when we have to bump z-indexes. Normally I try to keep z-indexes localized as much as possible, as in within a [single stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context), since otherwise it's really hard to figure out which z-index values are being assigned to components within the same stacking context but in very different files.

One approach I've tried in the past is to have a single Sass file define constants for all the global z-indexes, and then to have each component use their respective constant. E.g.

```scss
$z-index-site-nav: 3;
$z-index-filters-menu: 4;
```

This isn't a perfect solution, since there's no such thing as a global z-index (z-indexes are always scope to their stacking context), but it's at least better than having hardcoded numbers split across multiple files, since we'll have a single place where we can see the ordering of layers.

The real fix would be to restructure our HTML and CSS so that we don't have to reason about floating elements' z-indexes across multiple files, but that's going to be a bigger project.

Reply:
It's a tricky issue to solve for. At previous companies I've worked out, we used general vars which at least act as enums of sorts to limit all manner of z-index values. I like the specific var names you suggest and have created a couple of them in our layouts.scss file, which I think is a small improvement for now, short of a larger restructure of HTML/CSS